lib/Experian/IDAuth.pm

```diff
@@ -2,7 +2,7 @@ package Experian::IDAuth;
 use strict;
 use warnings;
 
-our $VERSION = '2.4';
```

Comment:
actually I have uploaded 2.5 (as dzil automatically increased it to 2.5) already to cpan, its waiting for upload.